presto-main/src/test/java/io/prestosql/type/TestRowOperators.java

```diff
@@ -481,7 +481,7 @@ public void testRowComparison()
         assertFunction("row(2, CAST(NULL AS INTEGER)) = row(1, 2)", BOOLEAN, false);
         assertFunction("row(2, CAST(NULL AS INTEGER)) != row(1, 2)", BOOLEAN, true);
         assertInvalidFunction("row(TRUE, ARRAY [1, 2], MAP(ARRAY[1, 3], ARRAY[2.0E0, 4.0E0])) > row(TRUE, ARRAY [1, 2], MAP(ARRAY[1, 3], ARRAY[2.0E0, 4.0E0]))",
-                TYPE_MISMATCH, "line 1:64: '>' cannot be applied to row(boolean, array(integer), map(integer, double)), row(boolean, array(integer), map(integer, double))");
+                TYPE_MISMATCH, "line 1:64: Operator does not exist: row(boolean, array(integer), map(integer, double)) > row(boolean, array(integer), map(integer, double))");
```

Comment:
Wasn't the original wording more readable?

Reply:
`integer = text` looks nice. For complex types (like the one here), it's hard to fish out the actual operator... maybe it's hard no matter what we do though.